core/include/tee/tee_svc_cryp.h

```diff
@@ -93,4 +93,9 @@ TEE_Result tee_obj_attr_from_binary(struct tee_obj *o, const void *data,
 				    size_t data_len);
 TEE_Result tee_obj_attr_copy_from(struct tee_obj *o, const struct tee_obj *src);
 
+struct bbuf_obj {
```

Comment:
suggestion:
`/* Helper to factorize bounce buffers allocation in key derivation functions */`

Also: I think this struct should be defined from _tee_svc_cryp.c_, not needed in this header file.

Reply:
Added the comment and moved the type declaration to *tee_svc_cryp.c*.